### ISSUE TITLE
Do not overwrite default failure transport

### ISSUE THREAD

**Issue by morgannilsson** [pimcore/generic-data-index-bundle#238]
generic-data-index-bundle/config/pimcore/messenger.yaml
```
framework:
    messenger:
        enabled: true
        failure_transport: pimcore_generic_data_index_failed

```
Should be:

```
framework:
    messenger:
        transports:
            pimcore_generic_data_index:
                dsn: "doctrine://default?queue_name=pimcore_generic_data_index_queue"
                failure_transport: pimcore_generic_data_index_failed
```

**Comment by morgannilsson** [comment]
PR created: https://github.com/pimcore/generic-data-index-bundle/pull/258

**Comment by morgannilsson** [comment]
New PR for 1.2 branch: [261](https://github.com/pimcore/generic-data-index-bundle/pull/261)

**Comment by mattamon** [comment]
Resolved by #261
